.travis.yml

```
@@ -15,5 +15,5 @@ deploy:
     skip_cleanup: true
     script: bash scripts/travis-deploy.sh
     on:
-      repo: dash
+      repo: @dashevo/dashjs
```

I guess this will be fixed by PR #18 probably
```suggestion
      repo: dashevo/DashJS
```